konfera/models/ticket.py

```diff
@@ -30,3 +32,14 @@ def __str__(self):
             first_name=self.first_name,
             last_name=self.last_name
         ).strip()
+
+    def save(self, *args, **kwargs):
+        if not hasattr(self, 'order'):
+            discount = 0
+            if self.discount_code:
+                discount = self.type.price * self.discount_code.discount/100
+            order = Order(price=self.type.price, discount=discount, status='awaiting_payment',
+                          purchase_date=timezone.now())
+            order.save()
+            self.order = order
+        super(Ticket, self).save(*args, **kwargs) # Call the "real" save() method.
```

Comment:
Comments are good, but in this case it is obvious what you are doing, therefore I think not necessary here. Besides you are missing one space there, so it breaks PEP8 and travis continuous deployment. Either fix PEP8 or remove comment.